### ISSUE TITLE
Cluster name has no environment variable

### ISSUE THREAD

**Issue by neilstevenson** [hazelcast/hazelcast-docker#123]
Docker image doesn't allow the cluster name to be set

So if the image is used to create a cluster for a Dev environment, and a cluster for a Test environment, both will have the group name "_dev_".

Confusing at least, and means one Management Center can't be used to manage such clusters,

Providing a custom `hazelcast.xml` is a workaround, but not as easy as an environment variable, especially for someone who understands Docker but not Hazelcast.

**Comment by leszko** [comment]
Yeah, there a lot of parameters that can't be set via env variables. I'd not make an exception for the cluster group name. Unless, it's really what people ask for all the time. Like we did an exception for Management Center URL and enabling SSL.

The good news is that the core team has already [PRD on making all parameters configurable from the command line](https://hazelcast.atlassian.net/wiki/spaces/PM/pages/1861189796/Allow+overriding+config+values+from+command+line). I think the right way of solving the root cause is to push on them to include it in the `4.1` roadmap.
